app/uk/gov/hmrc/individualsemploymentsapi/domain/v2/Employment.scala

```diff
@@ -32,9 +32,6 @@ case class Employment(
 )
 
 object Employment {
-
-  implicit val payFrequencyFormat: Format[PayFrequency] = EnumJson.enumFormat(PayFrequency)
-
   implicit val format: Format[Employment] = Format(
     (
```

Comment:
Any particular reason we are still doing explicit reads/writes for this model. Don't see any validation on fields for us to keep these explicit and can just use Format macro instead.

Reply:
Yes, it's rather frustrating but there is a field `payments` (plural) which is marshalled to/from `payment` (singular). I did not want to change it as it may break contracts

Reply:
it's small set of fields, so, fine to keep it. but, preferable/recommended way to handle those cases is to use transform combinators as that's what they are there for.

Reply:
actually, since that field gets marshalled to and from `payment`, there should be no issue in just renaming the field to `payment`and then using the format macro, i'll try this out and prepare another commit